priv/repo/structure.sql

```diff
@@ -1204,4 +1228,6 @@ INSERT INTO `schema_migrations` (version) VALUES (20220131103226);
 INSERT INTO `schema_migrations` (version) VALUES (20230217143550);
 INSERT INTO `schema_migrations` (version) VALUES (20230317094712);
 INSERT INTO `schema_migrations` (version) VALUES (20230402091100);
+INSERT INTO `schema_migrations` (version) VALUES (20230405111657);
```

Comment:
Most likely because it didn't change the schema (only a data migration):

https://github.com/instedd/surveda/blob/main/priv/repo/migrations/20230405111657_remove_project_memberships_duplicates.exs